src/main/java/org/folio/rest/impl/UsersAPI.java

```diff
@@ -544,4 +521,13 @@ private boolean checkForDuplicateAddressTypes(User user) {
     });
     return future;
   }
+
+  private Map<String, Object> getCustomFields(User entity) {
+    if(entity.getCustomFields()!= null){
+      return entity.getCustomFields().getAdditionalProperties();
+    }
+    else{
+      return Collections.emptyMap();
+    }
```

Comment:
Using a guard clause increases readability because the main code is no longer wrapped in the else clause (reduced indentation).

      if (entity.getCustomFields() == null) {
        return Collections.emptyMap();
      }
      return entity.getCustomFields().getAdditionalProperties();